### DISCUSSION TITLE
Would this work for raspberry pi zero? raspberry pi 3/4?

### DISCUSSION THREAD

**Discussion by talksik** [emilk/egui#3226] [Q&A]
I am unsure of the requirements of this. Does this require Opengl or something to render?

**Comment by MDZZDYXCMDZZDYXC** [comment]
> I am unsure of the requirements of this. Does this require Opengl or something to render?

You can just use egui and integrate your rendering, eframe depends on opengl or another rendering engine

**Comment by talksik** [comment]
Thank you for the response!

Sorry I'm a newbie when it comes to linux requirements for the pi. Do you have any resources I should look into on the steps I should take to get egui working on a pi?
